docs/source/user/aerodyn/theory_tailfin.rst

```diff
@@ -163,9 +161,24 @@ The user only needs to indicate the index `TFinAFIndex` within the list `AFNames
 Unsteady slender body model
 ---------------------------
 
-The unsteady slender body (USB) model is documented in :cite:`ad-hammam2022`.
+The unsteady aerodynamics of the tail fin is modeled based on Unsteady Slender Body Theory.
+The theory is extended to include the effect of high yaw angle :cite:`ad-hammam_NREL:2023`.
+
+The normal force on the tail fin can be described as
```

Comment:
can be described as the sum of three contributions (potential lift, vortex lift, and drag), weighted by separation functions `x_i` as: